app/src/controllers/enquiry.ts

```diff
@@ -4,7 +4,7 @@ import { generateCreateStamps, generateUpdateStamps } from '../db/utils/utils';
 import { activityService, enquiryService, noteService, userService } from '../services';
 import { Initiative } from '../utils/enums/application';
 import { ApplicationStatus, IntakeStatus, NoteType, SubmissionType } from '../utils/enums/housing';
-import { getCurrentIdentity, getCurrentTokenUsername } from '../utils/utils';
+import { getCurrentTokenSub, getCurrentTokenUsername } from '../utils/utils';
```

Comment:
Optional Style: It might be possible to rename to `getCurrentSubject` and `getCurrentUsername` as this information will always be inferred to have come from the JWT token.